src/transformers/hf_argparser.py

```diff
@@ -262,7 +274,9 @@ def parse_dict(self, args: Dict[str, Any], allow_extra_keys: bool = False) -> Tu
             outputs.append(obj)
         if not allow_extra_keys and unused_keys:
             raise ValueError(f"Some keys are not used by the HfArgumentParser: {sorted(unused_keys)}")
-        return tuple(outputs)
+        return tuple(
+            [_download_dataclass_zoo_stub_files(output) for output in outputs]
+        )
```

Comment:
not sure if i like this better but thought i'd add
```suggestion
        return tuple(map(_download_dataclass_zoo_stub_files, outputs))
```